controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/JobController.java

```diff
@@ -422,19 +422,15 @@ public void finish(RunId id) throws TimeoutException {
                 logs.flush(id);
                 metric.jobFinished(run.id().job(), finishedRun.status());
 
-                DeploymentId deploymentId = new DeploymentId(unlockedRun.id().application(), unlockedRun.id().job().type().zone(controller.system()));
```

Comment:
Ah, never mind this commit—I read that boolean inverted. This was correct after all. This was fixed later, when I rewrite this to also update the revision history.